src/middleware/queries.ts

```diff
@@ -354,6 +359,9 @@ export function transactions(
           success
           spec_version_id
           extrinsic_hash
+          block {
+            hash
```

Comment:
Same as above

Reply:
Yeah. `datetime` param is not used for this query